app/controllers/resources_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
   end
 
   def show
-    redirect_to(params.merge(:action => "edit"))
+    redirect_to :action => :edit
```

Comment:
Why is this a security issue?

On 2 March 2016 at 20:33, Zaid Akram <notifications@github.com> wrote:

> In app/controllers/resources_controller.rb
> <https://github.com/ari/jobsworth/pull/541#discussion_r54734263>:
>
> > @@ -13,7 +13,7 @@ def new
> >    end
> >
> >    def show
> > -    redirect_to(params.merge(:action => "edit"))
> > +    redirect_to :action => :edit
>
> Hence the security issue... Not sure which pramas we need in the edit
> action. I think instead of passing whole params hash we can filter using
> strong_parameters?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ari/jobsworth/pull/541/files#r54734263>.
>